thoth/common/logging.py

```diff
@@ -149,6 +149,28 @@ def _get_sentry_integrations() -> List[object]:
     return integrations
 
 
+def before_send_handler(event, hint):
+    """Filter the errors caught before sending to Sentry.
+
+    This function ignores the exceptions passed in as a environment variable in a comma separated manner.
+    """
+    ignored_exceptions = os.getenv('THOTH_SENTRY_IGNORE_EXCEPTION')
+    if not ignored_exceptions:
+        return event
+    exceptions_split = ignored_exceptions.split(',')
+    if 'exc_info' in hint:
+        exc_type, exc_value, tb = hint['exc_info']
+        for exception in exceptions_split:
+            if exception.strip() == exc_type.__name__:
```

Comment:
This operation will be done very often, so no need to do any additional operations like strip - the docs are explicitly saying how the string should be formatted.

Reply:
Okay will remove this.